common/CMakeLists.txt

```
@@ -218,6 +218,17 @@ if(APPLE)
 	target_link_options(common PRIVATE -fobjc-link-runtime)
 endif()
 
+if(UNIX AND NOT ANDROID AND NOT APPLE)
+	if(USE_DBUS)
```

```suggestion
if(DBUS_API)
```

and then in https://github.com/PCSX2/pcsx2/blob/388da2058b5de4db17103cca6e708cdc44127eb3/cmake/BuildParameters.cmake#L36

`option(DBUS_API "Enable DBus support for screensaver inhibiting" ON)`

(I changed the name for consistency with the other options.)

---

So it is that file that had the default build parameters :smile: